warp10/src/main/java/io/warp10/script/MemoryWarpScriptStack.java

```diff
@@ -1614,14 +1617,39 @@ private String rewriteMacroSymbol(String symbol) {
   }
   
   @Override
-  public void abort() {
+  public void signal(Signal signal) {
+    // Only set the signal if the stack is not yet signaled so a KILL is not
+    // overriden by anything else for example
+    if (this.signaled) {
+      return;
+    }
```

Comment:
We should still allow overriding a STOP by a KILL.

Reply:
So enforce a hierarchy of signals